mail_tracking/models/mail_tracking_email.py

```diff
@@ -162,7 +162,9 @@ def email_score(self):
     @api.depends('recipient')
     def _compute_recipient_address(self):
         for email in self:
-            if email.recipient:
+            is_empty_recipient = not email.recipient or re.search(
+                r'<False>', email.recipient)
```

Comment:
Do you really need a regexp for that? `'<False>' in email.recipient` should do the same...